docs/src/components/Utils/ApiTable.component.tsx

```diff
@@ -113,45 +116,75 @@ interface RowData {
     default?: string;
 }
 
-export const ApiTable = ({ data }: ApiTableProps): React.ReactElement<any> => (
-    <Table>
-        <thead>
-            <tr>
-                <th>Property</th>
-                <th>Description</th>
-                <th>Type</th>
-                <th>Default</th>
-            </tr>
-        </thead>
-        <tbody>
-            {data &&
-                data.map(
-                    (rowData: RowData): React.ReactElement<any> => (
-                        <tr key={`key-${rowData.property}`}>
-                            <td>{rowData.property}</td>
-                            <td>
-                                {Array.isArray(rowData.description)
-                                    ? rowData.description.map(
-                                          (description, i) => (
-                                              <React.Fragment key={`key-${i}`}>
-                                                  {typeof description ===
-                                                  'string'
-                                                      ? parse(description)
-                                                      : description}
-                                              </React.Fragment>
+const sortFn = (a: RowData, b: RowData) => {
```

Comment:
I feel like this function could be more descriptively named, or—since this is basically a simple sort method—if could just be an inline method inside of the view-partial.